source/cpp/source/DefaultCommandHandler.cpp

```diff
@@ -361,16 +362,16 @@ std::variant<juce::Slider *, juce::String> getSlider (const Command & command)
     if (component == nullptr)
         return "Component not found: " + componentId;
 
-    auto * slider = dynamic_cast<juce::Slider *> (component);
-    if (slider == nullptr)
-        return "Component is not a slider: " + componentId;
+    auto * castedComponenet = dynamic_cast<T *> (component);
+    if (castedComponenet == nullptr)
+        return "Component is not a " + juce::String(typeid(T).name()) + ": " + componentId;
 
-    return slider;
+    return castedComponenet;
```

Comment:
There is a spelling mistake here (`Componenet` should be `Component`).

Reply:
done